web/src/lib.rs

```diff
@@ -224,14 +223,24 @@ impl Ruffle {
     }
 
     pub fn pause(&mut self) {
-        // Remove instance from the active list.
         INSTANCES.with(|instances| {
             let instances = instances.borrow();
             let instance = instances.get(self.0).unwrap();
             instance.borrow().core.lock().unwrap().set_is_playing(false);
         });
     }
 
+    pub fn is_playing(&mut self) -> JsValue {
```

Comment:
You can return a standard rust `bool` here and `wasm_bindgen` takes care of converting it to JS:
```
        INSTANCES.with(|instances| {
            let instances = instances.borrow();
            let instance = instances.get(self.0).unwrap();
            let is_playing = instance.borrow().core.lock().unwrap().is_playing();
            is_playing
        })
```